### ISSUE TITLE
Telemetry app reload fails with 'CacheBytesAdded not found'

### ISSUE THREAD

**Issue by levi-turner** [eapowertools-archive/qs-telemetry-dashboard-node#82]
```
Drop field CacheBytesAdded from QIXPerformance_Engine
Error: Field 'CacheBytesAdded' not found
Execution Failed
Execution finished.
```

**Cause**: Engine > Logging > Performance set to Warning rather than Info (which is the default logging level).

**Comment by levi-turner** [comment]
Fixed on master (proposed 4.0 release)

**Comment by jparis** [comment]
fixed with new version of Telemetry. Official release coming very soon: https://github.com/eapowertools/qs-telemetry-dashboard
